source/help/getting-started/organizing-conversations.rst

```diff
@@ -154,7 +171,11 @@ When configured by the System Admin, users can enable the features in **Account
 
 **Collapsible custom categories**: Create custom categories in the sidebar to group channels together for easier navigation (e.g. “Design” or “Marketing”). Collapse categories to display only unread channels and reduce unnecessary scrolling.
 
-**Drag and drop channels and categories**: Drag channels between or within categories, or drag to reorder entire categories to prioritize important conversations.
+**Drag and drop channels and categories**: Drag channels between or within custom categories, or drag to reorder entire categories to prioritize important conversations.
+
+**Rename custom categories**: Customize name of categories.
```

Comment:
I wouldn't consider renaming at the same level of importance as the other things in this list. Renaming kind of goes without saying. This is more of a features highlight list in my mind

Reply:
Sounds good. Thanks! I've removed the bullet point for renaming.